model/src/main/proto/exploration.proto

```diff
@@ -322,6 +322,39 @@ message UserAnswer {
   WrittenTranslationContext written_translation_context = 6;
 }
 
+// Corresponds to a raw item selection answer that user has submitted.
+message ItemSelectionRawAnswer {
+  repeated int32 selected_indexes = 1;
+}
+
+// Corresponds to a raw user answer that user has submitted.
+message RawUserAnswer {
+  oneof answer_input_type {
+    // Just indicates the presence of the type.
+    bool continue = 1;
+    // A raw answer entered by user in TextInputInteraction.
+    string text = 2;
+    // A raw answer entered by user in FactionInputInteraction.
+    string fraction = 3;
+    // A raw answer entered by user in NumericInputInteraction.
+    string numeric = 4;
+    // A raw answer entered by user in NumericInputInteraction with units.
+    string number_with_units = 5;
+    // A raw answer entered by user in RatioInputInteraction.
+    string ratio_input = 6;
+    // A raw answer entered by user in MathInputInteraction.
+    string math_expression = 7;
```

Comment:
I'm actually wondering if we need to distinguish between these. We could, alternatively, just store a "textual_answer" that can be used by each of the interactions in turn.

Reply:
Updated to single textual_answer to store text based answers.